es_ES/_data/catalog/widgets.json

```diff
@@ -0,0 +1,1582 @@
+  },
+  {
+    "name": "Baseline",
+    "description": "Un widget que posiciona a su hijo de acuerdo con la línea base del hijo. ",
+    "categories": [
+    ],
+    "subcategories": [
+      "Single-child layout widgets"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/Baseline-class.html",
+    "image": "<svg viewBox='0 0 100 100'><defs><marker id='arrow-baseline' orient='auto-start-reverse' viewBox='0 0 1 1' markerWidth='3' markerHeight='3' refX='0.5' refY='0.5'><path d='M 1 0.5 L 0.5 0 L 0.5 1 z' fill='#ffffff'/></marker></defs><rect x='0' y='0' width='100' height='100' fill='#3949ab'/><rect x='20' y='20' width='60' height='50' fill='#ffffff'/><rect x='20' y='45' width='60' height='30' fill='#4dd0e1'/><line x1='85' y1='20' x2='85' y2='66' stroke='#ffffff' stroke-width='2' marker-end='url(#arrow-baseline)'/><line x1='15' y1='20' x2='15' y2='66' stroke='#ffffff' stroke-width='2' marker-end='url(#arrow-baseline)'/><line x1='10' y1='70' x2='90' y2='70' stroke='#ffffff' stroke-width='2'/><text x='50' y='69' text-anchor='middle' font-family='Roboto' font-size='25' fill='#3b75ad'>Abc </text></svg>"
+  },
+  {
+    "name": "FractionallySizedBox",
+    "description": "Un widget que dimensiona su hijo a una fracción del espacio total disponible. Para obtener más detalles sobre el algoritmo de diseño, consulte RenderFractionallySizedOverflowBox. ",
+    "categories": [
+    ],
+    "subcategories": [
+      "Single-child layout widgets"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/FractionallySizedBox-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "IntrinsicHeight",
+    "description": "Un widget que dimensiona su hijo a la altura intrínseca del hijo. ",
```

Comment:
A widget that sizes its child to the child's intrinsic height.
Un widget que dimensiona su hijo de acuerdo a la altura intrínseca de los hijos de este.